.github/workflows/up_swagger.sh

```diff
@@ -22,7 +22,7 @@ rm -rf node_modules
 
 # Same as https://github.com/swagger-api/swagger-ui/blob/63ad6f6a5bce19075e717ea74acaf9f7055dcdf5/docker/docker-entrypoint.d/40-swagger-ui.sh#L12
 FIND="\"https://petstore.swagger.io/v2/swagger.json\""
-REPLACE="window.location.origin + \"/api-docs/swagger.json\""
+REPLACE="window.location.origin + __SWAGGER_PATH__ + \'/swagger.json\'"
 sed -i -e "s|${FIND}|${REPLACE}|g" priv/swagger/swagger-initializer.js
```

Comment:
Shouldn't this generate `swagger-initializer.template` instead of `swagger-initializer.js`?
Furthermore, I would think this tool should _delete_ `swagger-initializer.js`

Reply:
And I would still prefer to allow the user to adjust the whole url and not just the path.
Remember that openapi's version of it is `https://petstore.swagger.io/v2/swagger.json`, after all.

Reply:
> Shouldn't this generate swagger-initializer.template instead of swagger-initializer.js?
Furthermore, I would think this tool should delete swagger-initializer.js

Actually it all depended if JS file will be replaced before or after compiling...

Reply:
Changed to `*.template`